examples/bq_file_load_benchmark/benchmark_tools/schema_creator.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import json
+import logging
+import os
+
+
+class SchemaCreator(object):
+    """Contains methods for creating schemas json schemas from file parameters.
+
+    Attributes:
+        schemas_dir(str): Directory where json schemas should be written to.
+        file_params(dict): Dictionary containing each file parameter and
+            its possible values.
+    """
+
+    def __init__(self, schemas_dir, file_params):
+        self.schemas_dir = schemas_dir
+        self.file_params = file_params
+
+    def create_schemas(self):
+        """Creates schemas in json format from file parameters.
+
+        Gathers combinations of schemas from the parameters in the numColumns
+        and columnTypes parameters in the self.file_params
+        dictionary. The schemas are written out in json format to the
+        self.schemas_dir directory. They can then be used to create staging
+        BigQuery tables.
+        """
+
+        # Gather file parameters.
+        num_columns = self.file_params['numColumns']
+        column_types = self.file_params['columnTypes']
+
+        # Begin iterating through parameters to create schemas.
+        for column_type in column_types:
```

Comment:
IMO this is a schema type as it specifies the type of schema to create, 
throughout this example we should refactor to do schema type.
As is it looks like you're writing a json file per column which is confusing.
In fact you're writing a json file per schema.